### ISSUE TITLE
[CI] EqlActionIT test failing reproducibly

### ISSUE THREAD

**Issue by mark-vieira** [elastic/elasticsearch#52737]
Just got a whole batch of failures for `EqlActionIT`. Example:

```
org.elasticsearch.xpack.eql.action.EqlActionIT > test {1.test} FAILED
    java.lang.AssertionError: no proxy found for action: org.elasticsearch.xpack.eql.action.EqlSearchAction@84757f79
        at __randomizedtesting.SeedInfo.seed([C92BF8205D60A3C7:417FC7FAF39CCE3F]:0)
        at org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:56)
        at org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:396)
        at org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
        at org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:385)
        at org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:45)
        at org.elasticsearch.action.ActionRequestBuilder.get(ActionRequestBuilder.java:52)
        at org.elasticsearch.xpack.eql.action.EqlActionIT.test(EqlActionIT.java:100)
REPRODUCE WITH: ./gradlew ':x-pack:plugin:eql:internalClusterTest' --tests "org.elasticsearch.xpack.eql.action.EqlActionIT" -Dtests.method="test {1.test}" -Dtests.seed=C92BF8205D60A3C7 -Dtests.security.manager=true -Dtests.locale=ar-IQ -Dtests.timezone=Etc/GMT-6 -Dcompiler.java=13
```

https://gradle-enterprise.elastic.co/s/erbbzrkrfrlzs/tests/t2lbc6rzxwpfa-wlpqa3kz72tuu

So far all the failures are in the `7.x` branch. The example above reproduces for me locally.

**Comment by mark-vieira** [comment]
Since this reproduces, and we have over a dozen failures already I've gone ahead and muted this in `7.x` with 025352f0a41a7f39249a12840db06c686f566b78.

**Comment by aleksmaus** [comment]
The issue was addressed with the backport PR https://github.com/elastic/elasticsearch/pull/52725.
Will open another PR to unmute the test.

**Comment by costin** [comment]
Closed by #52757
